### PR TITLE
Schedule task finished

### PR DESCRIPTION
**Pull request by MoreThanADC**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @MoreThanADC

**Comment by ziobron** [comment]
⏱ 3 XP extra for @MoreThanADC for on-time delivery

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @MoreThanADC

**Comment by ziobron** [comment]
⏱ 3 XP extra for @MoreThanADC for on-time delivery

**Review by lukasz-kukulka — APPROVED** [review]

**Comment by ziobron** [comment]
Thanks @lukasz-kukulka for Code Review.
🏅 1 XP granted

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#include <chrono>
+#include <functional>
+#include <thread>
+
+template<typename T, typename... Args>
+void schedule(T lazyFunction, std::chrono::seconds(duration), Args... args) {
+    std::this_thread::sleep_for(duration);
+    lazyFunction(args...);
```
Comment (nauka-programowania-MA) [inline homework/schedule/schedule.hpp:8]: use perfect forwarding here. std::forward<Args>(args)...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**